src/clients/SnapshotApi.ts

```diff
@@ -139,13 +139,15 @@ export class SnapshotApi {
     account: Web3Provider | Wallet,
     address: string,
     proposalSnapshotId: string,
-    choiceNumber: number
+    choiceNumber: number,
+    comment?: string
```

Comment:
Why don't we call this `reason`?

Reply:
<img width="229" alt="image" src="https://user-images.githubusercontent.com/2858950/202516313-c56d79e6-c506-4189-94ce-9cd85d4e4cb6.png">